data/changelog/odoc/2019-03-08-odoc-1.4.0.md

```diff
@@ -105,10 +105,10 @@ While release 1.4.0 has new features, it is mainly focused on stability and main
 
 In summary, compared with 1.2.0, odoc now has:
 
-- [New appearance](http://odis.io/odoc-demo/order/Order/index.html).
+- [New appearance](http://odis.io/odoc-demo/order/Order/index.html - [404 Not Found]).
 - Tables of contents on each page.
 - [Generation of docs in ReasonML syntax](https://github.com/ocaml/odoc/pull/156).
-- [Interop with BuckleScript](https://ostera.github.io/bsdoc/docs/BsDoc/).
+- [Interop with BuckleScript](https://ostera.github.io/bsdoc/docs/BsDoc/ - [404 Not Found]).
```

Comment:
```suggestion
- [Interop with BuckleScript](https://rescript-lang.org/blog/bucklescript-is-rebranding).
```
best think I could find about BuckleScript

Reply:
The project to search for was BsDoc.

```suggestion
- [Interop with BuckleScript](https://github.com/reasonml-community/bsdoc).
```